connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -183,7 +183,7 @@ object KustoWriter {
                        tableCoordinates: KustoCoordinates,
                        tmpTableName: String,
                        client: KustoClient): BlobWriteResource = {
-    val blobName = s"${tableCoordinates.database}_${tmpTableName}_${UUID.randomUUID.toString}_spark.csv.gz"
+    val blobName = s"${KustoQueryUtils.simplifyName(tableCoordinates.database)}_${tmpTableName}_${UUID.randomUUID.toString}_spark.csv.gz"
```

Comment:
This should be normalizeTableName, shouldn't it?
Also, if that function relates to any Kusto entity name, it should be named properly

Reply:
normalizeTableName doesn't remove whitesspaces and a blob name cannot have whitespaces